CODING_STYLE.md

```diff
@@ -0,0 +1,39 @@
+# Coding Style
+
+## General
+
+When contributing to the project, and if otherwise not mentioned in this document, our coding conventions
+follow the Microsoft [C# Coding Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions)
+and standard [Naming Guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines).
+
+## Class Members
+
+Members and types should always have the lowest possible visibility.
+
+Ordering of class members should be the following:
+1. Constants
+2. Fields
```

Comment:
I suggest to use build-in `1.` auto increment functionality for every item for better maintainability.
```
1. Constants
1. Fields
1. Abstract members
1. Constructors
1. Methods and Properties
1. Private nested classes

```